polkadot-parachains/statemine/src/constants.rs

```diff
@@ -14,15 +14,16 @@
 // limitations under the License.
 
 pub mod currency {
+	use kusama_runtime_constants as constants;
 	use node_primitives::Balance;
 
 	/// The existential deposit. Set to 1/10 of its parent Relay Chain (v9020).
-	pub const EXISTENTIAL_DEPOSIT: Balance = CENTS / 10;
+	pub const EXISTENTIAL_DEPOSIT: Balance = constants::currency::EXISTENTIAL_DEPOSIT / 10;
 
-	pub const UNITS: Balance = 1_000_000_000_000;
-	pub const CENTS: Balance = UNITS / 30_000;
-	pub const GRAND: Balance = CENTS * 100_000;
-	pub const MILLICENTS: Balance = CENTS / 1_000;
+	pub const UNITS: Balance = constants::currency::UNITS;
+	pub const CENTS: Balance = constants::currency::CENTS;
+	pub const GRAND: Balance = constants::currency::GRAND;
+	pub const MILLICENTS: Balance = constants::currency::MILLICENTS;
 
 	pub const fn deposit(items: u32, bytes: u32) -> Balance {
 		// map to 1/10 of what the kusama relay chain charges (v9020)
```

Comment:
would be good to use the kusama `deposit` function here (and divide by ten)
(and use the corresponding `deposit` functions in other runtimes respectively)

Reply:
That should do it :)